pkg/Microsoft.ML.DnnImageFeaturizer.AlexNet/Microsoft.ML.DnnImageFeaturizer.AlexNet.nupkgproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for pretrained AlexNet image featurization</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.nupkgproj" />
+  </ItemGroup>
+
+  <ItemGroup>  
+    <Content Include="$(LocalAppData)\mlnet-resources\AlexNetOnnx\AlexNet.onnx" Pack="true" PackagePath=".\lib\netstandard2.0\AlexNetOnnx" /> 
```

Comment:
These `.onnx` files shouldn't go under the `lib` folder in the NuGet package. The `lib` folder is only for .NET assemblies (more or less).

Instead, these files should be placed in the `contentFiles` directory.

See [https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-content-files](https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-content-files)

Reply:
Changed to go into contentFiles. Thanks!

---
In reply to: [232773780](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232773780) [](ancestors = 232773780)